scripts/build/TestPlatform.Dependencies.props

```diff
@@ -11,7 +11,7 @@
     <!-- this version also needs to be "statically" readable because the test fixture will inspect this file for the version
     and because during the test `dotnet test` will run and re-build some of the test projects and at that time the version
     from a build parameter would not be available, so I am writing this version from the build.ps1 script to keep it in sync -->
-    <NETTestSdkVersion>17.3.0-dev</NETTestSdkVersion>
+    <NETTestSdkVersion>17.4.0-dev</NETTestSdkVersion>
```

Comment:
This was automatically changed by build.cmd, I don't know if this needs to be kept or not.

Reply:
Yes. Main is branded with 17.4 now